.github/workflows/build-dev.yml

```diff
@@ -42,6 +42,7 @@ jobs:
     - name: Package extension
       if: github.event_name == 'push' && matrix.os == 'ubuntu-latest'
```

Comment:
@Tatsinnit I think npm run package does the same vsce package command which seems like a duplicate at line 46 and also the sample run from your fork didn't run the Package Extension step because it has to be github.event_name == 'push' - probably merging the PR will trigger this step to run. can you check on this please ?

Reply:
This fix is focused to fix the build at the moment. The Idea was to not touch too much of workflow for the fix, and revisit all this properly. aka whatever was the behaviour it will stay as it is.

Kind of want to avoid fix vs what's the pipeline doing, please check history / commit history and the relevance to properly revisit this workflow separately. cc: @hsubramanianaks

Reply:
@Tatsinnit thanks for the clarification (Sorry I am missing the history behind it), I thought you are fixing the vsce package which got removed in previous PR's. But if this is just to fix the build, then we are good.